front/app/utils/dateUtils.test.ts

```diff
@@ -77,7 +77,7 @@ describe('timeAgo is reported correctly', () => {
     let date = new Date();
     date.setMonth(date.getMonth() - 1);
     let timeAgoResponse = timeAgo(date.valueOf(), 'en') || '';
-    expect(timeAgoResponse).toEqual('1 month ago');
+    expect(timeAgoResponse).toEqual('4 weeks ago');
```

Comment:
I'm also wondering if we want to display "4 weeks" instead of 1 month. The name of test should change if we keep the weeks:

Name of test is now: "should accurately return months passed since a date"

Reply:
Sorry, should have posted before approving.

Reply:
I don't consider the name to be inaccurate now but feel free to create a ticket, if it bothers you.